src/Mod/OpenSCAD/InitGui.py

```diff
@@ -70,6 +67,7 @@ def QT_TRANSLATE_NOOP(scope, text):
             if openscadfilename: #automatic search was succsessful
                 FreeCAD.addImportType("OpenSCAD Format (*.scad)","importCSG") 
                 param.SetString('openscadexecutable',openscadfilename) #save the result
+                FreeCAD.__unit_test__ += ["TestOpenSCADGui"]
```

Comment:
I don't understand why this line was moved here. Could you explain why we the unit test only should be added if `param.GetString('openscadexecutable')` is empty?

Also there's one `import OpenSCADUtils` too much, either the one just above or the one at the top of the file.

Reply:
Actually, I'd say that whole conditional is probably indented one too many times. I'm not sure why the addImportType exists here at all, to be honest, this is the GUI code, and presumably that import doesn't require the GUI.

Reply:
Fixed now, thanks. I'll submit a separate PR to deal with the non-storage of the executable in the main module.